types/index.esm.d.ts

```diff
@@ -594,24 +594,23 @@ export interface DatasetControllerChartComponent extends ChartComponent {
 }
 
 export type AnyObject = Record<string, unknown>;
-export interface Defaults extends CoreChartOptions, ElementChartOptions {
+export interface Defaults extends CoreChartOptions, ElementChartOptions, PluginChartOptions {
 	controllers: {
 		[key in ChartType]: DeepPartial<
 			CoreChartOptions &
 			ElementChartOptions &
+			PluginChartOptions &
 			DatasetChartOptions<key>[key] &
 			ScaleChartOptions<key> &
 			ChartTypeRegistry[key]['chartOptions']
 			>;
 	};
 
-	scale: ScaleOptions;
+	scale: ScaleOptionsByType;
 	scales: {
-		[key in ScaleType]: ScaleOptions<key>;
+		[key in ScaleType]: ScaleOptionsByType<key>;
```

Comment:
Actually, this one could also be `scales: Partial<ScaleOptionsByType>`

Reply:
Though, I have a doubt.

Reply:
I will look at that one separately since it needs more thought. `Partial` allows undefined but in the defaults they may be required.